test/Tests/src/Data/Vector_Spec.enso

```diff
@@ -223,7 +223,6 @@ spec = Test.group "Vectors" <|
         ["abab", "aaabaaaa"].filter (Filter_Condition.Like "_ba_") . should_equal ["abab"]
         ["abab", "aaabaaaa"].filter (Filter_Condition.Like "%ba__%") . should_equal ["aaabaaaa"]
         ["aaaa", "bbbbb", "[ab]aaaa"].filter (Filter_Condition.Like "[ab]%") . should_equal ["[ab]aaaa"]
-        ["a\Qa\Eabb", "aaabb"].filter (Filter_Condition.Like "_\Qa\Ea%") . should_equal ["a\Qa\Eabb"]
         ["f.txt", "abc.*"].filter (Filter_Condition.Like "%.*") . should_equal ["abc.*"]
```

Comment:
why was this test removed?

Reply:
\Q and \E escape all regex special characters between them, which is supported by Java regex, but not by Truffle regexp. That wouldn't affect this test, but it's misleading.

Reply:
I'd keep it. Why is it misleading? It checks that `Like` is not using Regex syntax but the SQL one.

Rather than removing, I'd add a comment to it explaining - to avoid any confusion.

Reply:
It is an invalid escape and so the regex compile step fails.